tailcfg/tailcfg.go

```diff
@@ -920,6 +920,12 @@ type PingRequest struct {
 	// Log is whether to log about this ping in the success case.
 	// For failure cases, the client will log regardless.
 	Log bool `json:",omitempty"`
+
+	// Types is the types of ping that is initiated. Can be TSMP, ICMP or DISCO
```

Comment:
singular or plural? if plural, how? comma-separated, or?

Also, DISCO isn't an acronym. And missing final period.